### DISCUSSION TITLE
A5006779-694A-1139-D643-08BDCD0B875E

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#174] [General]
# A5006779-694A-1139-D643-08BDCD0B875E

This addon adds Comet C/2021 A1 (Leonard), which was a comet visible from Earth between December 2021 and January 2022.

https://celestia.mobi/addon?item=A5006779-694A-1139-D643-08BDCD0B875E&lang=es

**Comment by SiriusBislifeless** [comment]
bro it has my name
